lib/fastlane/actions/podio_item.rb

```diff
@@ -0,0 +1,215 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      PODIO_ITEM_URL = :PODIO_ITEM_URL
+    end
+
+    class PodioItemAction < Action
+      AUTH_URL = "https://podio.com/oauth/token"
+      BASE_URL = "https://api.podio.com"
+
+      def self.run(params)
+        Helper.log.info "Parameter Client ID: #{params[:client_id]}"
+        Helper.log.info "Parameter Client secret: #{params[:client_secret]}"
+        Helper.log.info "Parameter App ID: #{params[:app_id]}"
```

Comment:
Do you really want to print out all these values? I don't think that's necessary